src/rewrite_rnn.cpp

```diff
@@ -1166,5 +1167,13 @@ std::vector<operation> rewrite_rnn::lstm_actv_funcs(instruction_ref ins) const
     }
 }
 
+namespace op {
+std::ostream& operator<<(std::ostream& os, rnn_direction v)
+{
+    os << static_cast<std::underlying_type<rnn_direction>::type>(v);
```

Comment:
This should print the enum name not number to make it clear for output.

Reply:
fixed